eggplant/profiles/views.py

```diff
@@ -103,12 +105,17 @@ def get_initial(self):
 
     def form_valid(self, form):
         user_id = self.request.user.id
+        photo = self.request.FILES.get('photo')
```

Comment:
I get an InMemoryUploadedFile here but can't seem to save it on the UserProfile below. Any ideas?